pkg/operator/ceph/pool/controller_test.go

```diff
@@ -562,7 +562,7 @@ func TestCephBlockPoolController(t *testing.T) {
 		err = r.client.Get(context.TODO(), req.NamespacedName, pool)
 		assert.NoError(t, err)
 		assert.Equal(t, cephv1.ConditionReady, pool.Status.Phase)
-		assert.Nil(t, pool.Status.MirroringStatus)
+		assert.Equal(t, pool.Status.MirroringStatus, &cephv1.MirroringStatusSpec{})
```

Comment:
@travisn We can decide, whether to show empty spec in the status, or just not to show, if its disabled

Reply:
Would it make sense to set "disabled" for the status message? Or empty is probably fine.

Reply:
I am okay with both the options, currently with the changes it is empty struct,
So, are we concluding to keep it empty when the mirroring is disabled?